wiki/Game_mode/en.md

```diff
@@ -48,6 +50,12 @@ In *osu!catch*, fruits of differing varieties fall from the top of the screen, a
 
 ![Gameplay of osu!mania](/wiki/shared/mania-gameplay.jpg "osu!mania gameplay")
 
+## Custom game modes
+
+In osu!(lazer), you can use custom game modes (also called custom rulesets) that introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.
```

Comment:
Forgot that "you" should be avoided: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#perspective